Tests from head seem to not work.

Clone a fresh repo, and both on linux / mac, I get these two tests failing:
```
I0403 08:32:08.328275    4719 report.go:235] keeping: ref/path/lib64 - elf_lib_dir_refs - $ref: /lib64/ld-linux-x86-64.so.2
--- FAIL: TestSimple (0.16s)
    --- FAIL: TestSimple/Python/valryian_debug_setup.py (0.00s)
        bincapz_test.go:129: unexpected diff:   (
              	"""
              	# testdata/Python/valryian_debug_setup.py
            - 	combo/backdoor/py_setuptools
              	combo/recon/system_network
              	exec/pipe
              	... // 13 identical lines
              	"""
              )
```

```
--- FAIL: TestDiff (3.58s)
    --- FAIL: TestDiff/macOS/libffmpeg.dirty.mdiff (0.31s)
        bincapz_test.go:186: unexpected diff:   strings.Join({
              	"## Changed: ",
            + 	"testdata/macOS/libffmpeg.dirty",
              	".",
            + 	"dylib",
              	"\n\nPrevious Risk: ✅ 2/MEDIUM\nNew Risk:      🚨 4/CRITICAL\n\n| ",
              	"   RISK     |                     KEY                      |    ",
              	... // 4345 identical bytes
              }, "")
```

--- a/bincapz_test.go
+++ b/bincapz_test.go
@@ -85,7 +85,7 @@ func TestSimple(t *testing.T) {
 
 	fs.WalkDir(fileSystem, ".", func(path string, d fs.DirEntry, err error) error {
 		if err != nil {
-			log.Fatal(err)
+			t.Fatal(err)
 		}
 		if !strings.HasSuffix(path, ".simple") {
 			return nil

--- a/testdata/Python/valyrian_debug_setup.py
+++ b/testdata/Python/valyrian_debug_setup.py
@@ -1,4 +1,4 @@
-# based on valryian_debug-0.0.1 from the backstabbers knife collection
+# based on valyrian_debug-0.0.1 from the backstabbers knife collection
 import json
 import os
 import sys

--- a/testdata/Python/valyrian_debug_setup.py.simple
+++ b/testdata/Python/valyrian_debug_setup.py.simple
@@ -1,4 +1,5 @@
-# testdata/Python/valryian_debug_setup.py
+# testdata/Python/valyrian_debug_setup.py
+combo/backdoor/py_setuptools
 combo/recon/system_network
 exec/pipe
 exec/program

--- a/testdata/macOS/libffmpeg.dirty.mdiff
+++ b/testdata/macOS/libffmpeg.dirty.mdiff
@@ -1,4 +1,4 @@
-## Changed: testdata/macOS/libffmpeg.dirty.dylib
+## Changed: .
 
 Previous Risk: ✅ 2/MEDIUM
 New Risk:      🚨 4/CRITICAL

